filters/color-overlay/src/ColorOverlayFilter.js

```diff
@@ -0,0 +1,54 @@
+import {vertex} from '@tools/fragments';
+import fragment from './colorOverlay.frag';
+import {Filter} from '@pixi/core';
+import {hex2rgb, rgb2hex} from '@pixi/utils';
+
+/**
+ * @class
```

Comment:
Please provide a description and link to the screenshot like [other filters](https://github.com/pixijs/pixi-filters/blob/master/filters/ascii/src/AsciiFilter.js#L14) so it shows up in the documentation.